### PR TITLE
VY-8 CRUD REST API for Blood groups 

### PR DESCRIPTION
**Pull request by lf-kiranpariyar**
@joshianuj @bishalshr @pratishshr please review the code

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -62,14 +73,21 @@
     "webpack-stream": "^3.1.0"
   },
   "dependencies": {
+    "axios": "^0.9.1",
```
Comment (pratishshr) [inline frontend/package.json]: which are you using?

File: frontend/package.json [hunk 2]
```diff
@@ -18,13 +19,17 @@
...
   "devDependencies": {
     "axios": "^0.9.1",
+    "babel-cli": "^6.9.0",
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babelify": "^7.2.0",
     "browser-sync": "^2.11.2",
     "browserify": "^13.0.0",
+    "chai": "^3.5.0",
```
Comment (pratishshr) [inline frontend/package.json]: ??

File: frontend/src/css/main.css [hunk 3]
```diff
@@ -1012,112 +1183,438 @@ li.dropdown-header {
     background-color: #f9f9f9;
 }
 
+/* Timeline */
+.timeline {
```
Comment (bishalshr) [inline frontend/src/css/main.css]: timeline????

File: frontend/src/js/components/blood-group/BloodGroupForm.js [hunk 4]
```diff
@@ -3,3 +3,136 @@
...
+
+class BloodGroupForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.saveBloodGroup = this.saveBloodGroup.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.state = {
+      autoFocus: true,
+      formValidator: new formValidator()
```
Comment (joshianuj) [inline frontend/src/js/components/blood-group/BloodGroupForm.js]: Use capital casing

File: frontend/src/js/constants/messageConstants.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+ * 5/19/16
+ */
+
+const messageConstants = {
+  FORM_INVALID_SUBMISSION_MESSAGE: 'Please fill the required fields with correct data',
+  FOREIGN_KEY_VIOLATION_MESSAGE: 'violates foreign key constraint',
+  UNSUCCESSFUL_DELETE_MESSAGE: 'Sorry, this field cannot be deleted',
+  DELETE_MESSAGE: 'Are you sure you want to delete this item?',
+
+  REQUIRED_MESSAGE: 'This field is required',
```
Comment (joshianuj) [inline frontend/src/js/constants/messageConstants.js]: end message with fullstop

**Pull request merged**